### PR TITLE
Reviewing plot bivariate

### PR DESCRIPTION
**Pull request by jrosen48**

### CODE REVIEW

**Comment by jrosen48** [comment]
Two related questions @cjvanlissa:

- When I pulled master, I couldn't access the documentation for `plot_bivariate()` (i.e., `?plot_bivariate())` didn't return anything), nor did it render a page when I ran `pkgdown::build_site()`
- When I ran `devtools::document()` to regenerate the docs, I got this pretty (j/p) set of messages:

```{r}
Updating tidyLPA documentation
Updating roxygen version in /Users/jrosenb8/tidyLPA/DESCRIPTION
Writing NAMESPACE
Loading tidyLPA
tidyLPA is intended for academic use. We do not make any money on this and only ask that you please cite this in publications when you use the results. You can use the function citation('tidyLPA') to create a citation.Mplus is not installed. Use only package = 'mclust' when calling estimate_profiles().
Error in get(name, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'wrapString' not found
Warning: [TEMPLATE:2] @usage requires a value
Error: Result 18 must be a single string, not NULL of length 0
Run `rlang::last_error()` to see where the error occurred.
In addition: Warning message:
In structure(x, class = "rd") :
  Calling 'structure(NULL, *)' is deprecated, as NULL cannot have attributes.
  Consider 'structure(list(), *)' instead.
```

I thought I may have goofed on something, as I accidentally made some commits to master instead of this branch, though I believe I've reverted those three, so the repo should be as it was when the branch for the PR for the plot bivariate functionality (#141) was merged. Sorry if I goofed on anything (and especially sorry for being so negligent in my reviewing).

**Comment by cjvanlissa** [comment]
When I look at your latest 9 commits, it seems like you deleted most of my work from the past months (see here https://github.com/cjvanlissa/tidyLPA/compare/master...data-edu:master). I think it would be better to go back to the latest version that I pushed.

**Comment by cjvanlissa** [comment]
This commit number: 791b33d15b8eb9ba05424cb94fbd87d45a92c32c

**Comment by jrosen48** [comment]
Kk, let me try again. I think I must have made the branch from a master which was not up-to-date.

**Comment by jrosen48** [comment]
I'm gonna ditch this branch and start over
